packages/headless/src/controllers/insight/search-box/headless-insight-search-box.ts

```diff
@@ -45,6 +45,7 @@ export function buildSearchBox(
     ...props,
     executeSearchActionCreator: executeSearch,
     fetchQuerySuggestionsActionCreator: fetchQuerySuggestions,
+    isNextAnalyticsReady: false,
```

Comment:
This allow the branching to work in the headlessCoreSearchBox.
When Insight migrate will have done the as this PR, they will be able to turn it to true (and we'll remove false and clean up)